dnsprovider/google.go

```diff
@@ -299,3 +306,7 @@ func newRecord(endpoint endpoint.Endpoint) *dns.ResourceRecordSet {
 func isNotFound(err error) bool {
 	return strings.Contains(err.Error(), "notFound")
 }
+
+func trackAPICall() {
+	metrics.APICallsTotal.WithLabelValues("google").Inc()
```

Comment:
have a global counter and counter per "method", then we would see flapping create/delete